src/wasm-validator.h

```diff
@@ -38,12 +38,26 @@
 #define wasm_wasm_validator_h
 
 #include <set>
+#include <sstream>
 
 #include "wasm.h"
 #include "wasm-printing.h"
 
 namespace wasm {
 
+// Print anything that can be streamed to an ostream
```

Comment:
I thought about that but they are kind of specific to the validator where we want to easily make everything print with the extra type info. Otherwise they sort of just duplicate the stream-related functions there.